var/spack/repos/builtin/packages/mozjs/package.py

```diff
@@ -31,15 +31,24 @@ class Mozjs(AutotoolsPackage):
     configure_directory = 'js/src'
     build_directory = 'js/src/spack-build'
 
-    patch('perl-bug.patch')
+    patch('perl-bug.patch', sha256='9f7d8502d85a4125e975a84cae11a8b34e696172d56f8ebc7ecf6d21fa3c30c9')
+    patch('Bug-638056-Avoid-The-cacheFlush-support-is-missing-o.patch',
```

Comment:
Could you add a comment that links to the relevant bug report? It looks like that is https://bugzilla.mozilla.org/show_bug.cgi?id=638056

Reply:
fixed in https://github.com/spack/spack/pull/17594/commits/24e62fd4ca57b695cf80754394454d0e173dcae9